demo/README.md

```diff
@@ -1,56 +0,0 @@
-# Summary
-This is a demo LabKey ReactJS module. This module demonstrates how to render a React application/page within the LabKey framework 
-including keeping the standard LabKey header and header menus.
```

Comment:
nit: A few references to the `demo` module path still exist in the top-level `.gitignore`. Consider removing.

Reply:
Thanks... removed